plugin/adsk/scripts/mayaUsd_pluginUICreation.mel

```diff
@@ -0,0 +1,15 @@
+global proc mayaUsd_pluginUICreation()
+// Callback registered with the MFnPlugin for when the plugin is loaded.
+// Used to initialize any UI we create.
+{
+    source "mayaUsdMenu.mel";
+    mayaUsdMenu_loadui;
+
+    int $mjv = `about -majorVersion`;
+
+    if (((2021 <= $mjv)) && (!`pluginInfo -q -loaded "ufeSupport"`)) {
+        if (catchQuiet(`loadPlugin "ufeSupport"`)) {

```

Comment:
I don't think we have an easy way to determine in a MEL script if we've been built with UFE support or not, but I don't think this is a big deal for the ufeSupport plugin.  It's pretty much a standalone thing, and it's currently autoloaded by Maya, so it should not be a bad thing if we try to load it from a MayaUSD plugin that someone built without UFE disabled.
The change I made here is to at least check for a version of Maya starting at 2021, and to spit out a warning if there was a problem loading it.